No video on call after update on v15.1.1

### Steps to reproduce

Try to call with video to other mobile

### Expected behaviour

If i'm calling no incoming video on my device (S10e)

### Actual behaviour

I'm revert on 15.1 version and all works fine.

### Device brand and model

Samsung S10e

### Android version

10

### Nextcloud Talk app version

16.0.0

### Nextcloud server version

26.0.1

### Talk version

17.0.0RC4

### Custom Signaling server configured

Yes (specify version in Additional Information)

### Custom TURN server configured

Yes

### Custom STUN server configured

Yes

### Android logs

_No response_

### Server log

_No response_

### Additional information

Janus ver. 1.1.4
Coturn ver. 4.6.2
Nats ver. 2.9.12
Signaling ver. 1.1.2

Just tested this : https://github.com/nextcloud/talk-android/pull/3177

made one call using two samsung android devices still same problem
<details>
<summary>Signaling log</summary>
<br>
Jul 14 20:39:29 turn systemd[1]: Started Nextcloud Talk signaling server. Jul 14 20:39:29 turn signaling[230990]: main.go:133: Starting up version /go1.20.5 as pid 230990 Jul 14 20:39:29 turn signaling[230990]: main.go:142: Using a maximum of 2 CPUs Jul 14 20:39:29 turn signaling[230990]: natsclient.go:108: Connection established to nats://127.0.0.1:4222 (NBGFOCH46X34NWZGN72TFNHMZCGFBAJURWZKH354AZ2UF7Q6TO4USHJB) Jul 14 20:39:29 turn signaling[230990]: grpc_common.go:167: WARNING: No GRPC server certificate and/or key configured, running unencrypted Jul 14 20:39:29 turn signaling[230990]: grpc_common.go:169: WARNING: No GRPC CA configured, expecting unencrypted connections Jul 14 20:39:29 turn signaling[230990]: backend_storage_static.go:72: Backend backend-1 added for https://cloud.wh-computers.co.uk/ Jul 14 20:39:29 turn signaling[230990]: hub.go:200: Using a maximum of 8 concurrent backend connections per host Jul 14 20:39:29 turn signaling[230990]: hub.go:207: Using a timeout of 10s for backend connections Jul 14 20:39:29 turn signaling[230990]: hub.go:303: Not using GeoIP database Jul 14 20:39:29 turn signaling[230990]: mcu_janus.go:294: Connected to Janus WebRTC Server 0.14.0 by Meetecho s.r.l. Jul 14 20:39:29 turn signaling[230990]: mcu_janus.go:300: Found JANUS VideoRoom plugin 0.0.9 by Meetecho s.r.l. Jul 14 20:39:29 turn signaling[230990]: mcu_janus.go:305: Data channels are supported Jul 14 20:39:29 turn signaling[230990]: mcu_janus.go:309: Full-Trickle is enabled Jul 14 20:39:29 turn signaling[230990]: mcu_janus.go:311: Maximum bandwidth 1048576 bits/sec per publishing stream Jul 14 20:39:29 turn signaling[230990]: mcu_janus.go:312: Maximum bandwidth 2097152 bits/sec per screensharing stream Jul 14 20:39:29 turn signaling[230990]: mcu_janus.go:318: Created Janus session 1119253239952861 Jul 14 20:39:29 turn signaling[230990]: mcu_janus.go:325: Created Janus handle 3996984912592662 Jul 14 20:39:29 turn signaling[230990]: main.go:263: Using janus MCU Jul 14 20:39:29 turn signaling[230990]: hub.go:385: Using a timeout of 10s for MCU requests Jul 14 20:39:29 turn signaling[230990]: backend_server.go:95: Using configured TURN API key Jul 14 20:39:29 turn signaling[230990]: backend_server.go:96: Using configured shared TURN secret Jul 14 20:39:29 turn signaling[230990]: backend_server.go:98: Adding "turn:127.0.0.1:3478?transport=udp" as TURN server Jul 14 20:39:29 turn signaling[230990]: backend_server.go:98: Adding "turn:127.0.0.1:3478?transport=tcp" as TURN server Jul 14 20:39:29 turn signaling[230990]: backend_server.go:111: No IPs configured for the stats endpoint, only allowing access from 127.0.0.1 Jul 14 20:39:29 turn signaling[230990]: main.go:339: Listening on 127.0.0.1:8080 Jul 14 20:39:29 turn signaling[230990]: client.go:284: Client from 192.168.1.203 has RTT of 14 ms (14.586708ms) Jul 14 20:39:30 turn signaling[230990]: capabilities.go:151: Capabilities expired for https://cloud.wh-computers.co.uk/ocs/v2.php/cloud/capabilities, updating Jul 14 20:39:30 turn signaling[230990]: client.go:284: Client from 192.168.1.203 has RTT of 60 ms (60.63233ms) Jul 14 20:39:30 turn signaling[230990]: capabilities.go:214: Received capabilities map[config:map[attachments:map[allowed:false] call:map[breakout-rooms:true can-upload-background:false enabled:true predefined-backgrounds:[1_office.jpg 2_home.jpg 3_abstract.jpg 4_beach.jpg 5_park.jpg 6_theater.jpg 7_library.jpg 8_space_station.jpg] recording:false supported-reactions:[ ]] chat:map[max-length:32000 read-privacy:0 translations:[] typing-privacy:0] conversations:map[can-create:false] previews:map[max-gif-size:3.145728e+06] signaling:map[hello-v2-token-key:-----BEGIN PUBLIC KEY----- Jul 14 20:39:30 turn signaling[230990]: MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEVKD0oL/yvpEzIxJAsKKRwzGUKkvo Jul 14 20:39:30 turn signaling[230990]: nLc7qH9x4xKtAeJj14x3XplFBXlFU1l5WZBEPvicCoTrLa/guCZX8wGxdg== Jul 14 20:39:30 turn signaling[230990]: -----END PUBLIC KEY----- Jul 14 20:39:30 turn signaling[230990]: session-ping-limit:200]] features:[audio video chat-v2 conversation-v4 guest-signaling empty-group-room guest-display-names multi-room-users favorites last-room-activity no-ping system-messages delete-messages mention-flag in-call-flags conversation-call-flags notification-levels invite-groups-and-mails locked-one-to-one-rooms read-only-rooms listable-rooms chat-read-marker chat-unread webinary-lobby start-call-flag chat-replies circles-support force-mute sip-support sip-support-nopin chat-read-status phonebook-search raise-hand room-description rich-object-sharing temp-user-avatar-api geo-location-sharing voice-message-sharing signaling-v3 publishing-permissions clear-history direct-mention-flag notification-calls conversation-permissions rich-object-list-media rich-object-delete unified-search chat-permission silent-send silent-call send-call-notification talk-polls breakout-rooms-v1 recording-v1 avatar chat-get-context single-conversation-status chat-keep-notifications typing-privacy message-expiration reactions chat-reference-id] version:17.0.1] from https://cloud.wh-computers.co.uk/ocs/v2.php/cloud/capabilities Jul 14 20:39:30 turn signaling[230990]: hub.go:740: Register user wayne@backend-1 from 192.168.1.203 in unknown-country (Mozilla/5.0 (Android) Nextcloud-Talk v3177) s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== (private=MTY4OTM2MzU3MHw1RUhkVGNCdTFTNkxGbFFnUVZqZExqZkNtR044SUdxRFJ6UTlOUGtZMWtsMUtFaU5TWE84WjJZQU9mbURvQ1VWNEtrTG1mNmtYMDVrTDBvT2xRZkRfQmxrVWpBRXl0S09TdnBHcnY0OGxwZElJeVNwd3Q0N1hyY1lBcEdTZG9YY0RDcndBOEJ0aENZNERQNElZVUU4aTloSVE0X0MtMmd0SlNfYUdPZ2NLQThtT1E9PXzZ6iQ0OozEgjQkso-AJOR9bmUROE0vDRfCpBxfKSndhg==) Jul 14 20:39:32 **turn signaling[230990]: client.go:303: Error reading from 192.168.1.203: read tcp 127.0.0.1:8080->127.0.0.1:46948: use of closed network connection** Jul 14 20:39:52 turn signaling[230990]: clientsession.go:434: Session s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== joined room 77j45iw7 with room session id pTjb3prXdrJk4DgnuWUMrQR4o9hPbmFJ7MsvWeNesuaN7zihRJoKuu69ymqtnh0LTCxHnzuGsbx1HgfvCV2zY6fNmoMrExDdh3ulF/gVS7yfByPqZXq2X43eee45zMIkzs/lxKa3z1ZCfW4g1lvxvn+cxOGbYOrsUM8tg9PPhMuQs2ZJRo80m255rTXcKgfBcP18vvdHcr6E93hYSjX0MNqt/qlxG2mRIcrUEsZ9yrkBegLeXwaDmApOwaohyb1 Jul 14 20:39:52 turn signaling[230990]: clientsession.go:286: Permissions of session s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== changed: [publish-audio publish-video publish-screen control] Jul 14 20:39:57 turn signaling[230990]: room.go:656: Session s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== joined call 77j45iw7 Jul 14 20:39:57 turn signaling[230990]: mcu_janus.go:729: Attached video as publisher 4982243268801941 to plugin janus.plugin.videoroom in session 1119253239952861 Jul 14 20:39:57 turn signaling[230990]: mcu_janus.go:767: Created room 7752095335139882 {janus.plugin.videoroom map[permanent:false room:7752095335139882 videoroom:created]} Jul 14 20:39:57 turn signaling[230990]: mcu_janus.go:825: Publisher s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== is using handle 4982243268801941 Jul 14 20:39:57 turn signaling[230990]: clientsession.go:890: Publishing video as 1 for session s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== Jul 14 20:39:57 turn signaling[230990]: mcu_janus.go:864: Publisher 4982243268801941 received connected Jul 14 20:39:59 turn signaling[230990]: hub.go:740: Register user **** from 192.168.1.43 in unknown-country (Mozilla/5.0 (Android) Nextcloud-Talk v17.0.1) Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== (private=MTY4OTM2MzU5OXxUTEZSbUxRRDZtSXdxRHZBR21Nb3VhYWtWb3ZiUTRxZTFneHFudW0yT3ZDMXZzNlF0SUFUOW9sMVZsSUV1ek9iaUU4ejFHRmVfVUtkV0xkUjdhNVlUem1icGRIVmpVS1BJcGMyVmJvYUlwZ283b2RPcVFkUkxkWnlXU09qUUE3VVRNNzVnZUY4YlNUQ1lhRG0zX3VrRFF5b1hsNjMxN0Q2N0pHNFR2dW1KaW51NXc9PXxXlFnlMRXh0_0qwDo_nV6zJsgsOPR-NhngLg4vW-R5MQ==) Jul 14 20:39:59 turn signaling[230990]: client.go:282: Client Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== has RTT of 92 ms (92.820028ms) Jul 14 20:40:03 turn signaling[230990]: clientsession.go:434: Session Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== joined room 77j45iw7 with room session id /AEiw2Bp+w7H8jjX78lCNU+3jVGOJF+u7TH3YLEfIN3Fs2m2HxGy3cDD8IsDI410Pk2Yu6D1rGeRhwlP3Dm8djtD4xc1vurRL8YgkC9NRNARI9Y5sztXiUPOHa8KEPxEMyVffixJhhXgHVS52hs6DaBsk8Dyw28/c6vbUM9JO7vUCt0miMR0WX6GQcK/noRUy8QE2T1MqoYakeEUXY1tQAYQuJ6UoE1/ukSWlYNs0E5B+YpNnYno0U+hqGi7cfW Jul 14 20:40:03 turn signaling[230990]: clientsession.go:286: Permissions of session Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== changed: [publish-audio publish-video publish-screen control] Jul 14 20:40:03 turn signaling[230990]: room.go:656: Session Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== joined call 77j45iw7 Jul 14 20:40:03 turn signaling[230990]: mcu_janus.go:729: Attached video as publisher 4012031908138812 to plugin janus.plugin.videoroom in session 1119253239952861 Jul 14 20:40:03 turn signaling[230990]: mcu_janus.go:767: Created room 5938377545275208 {janus.plugin.videoroom map[permanent:false room:5938377545275208 videoroom:created]} Jul 14 20:40:03 turn signaling[230990]: mcu_janus.go:825: Publisher Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== is using handle 4012031908138812 Jul 14 20:40:03 turn signaling[230990]: clientsession.go:890: Publishing video as 2 for session Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== Jul 14 20:40:03 turn signaling[230990]: mcu_janus.go:1024: Attached subscriber to room 7752095335139882 of publisher s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== in plugin janus.plugin.videoroom in session 1119253239952861 as 2171577364885873 Jul 14 20:40:03 turn signaling[230990]: clientsession.go:967: Subscribing video from s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== as 3 in session Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== Jul 14 20:40:03 turn signaling[230990]: mcu_janus.go:1024: Attached subscriber to room 5938377545275208 of publisher Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== in plugin janus.plugin.videoroom in session 1119253239952861 as 1602128575696101 Jul 14 20:40:03 turn signaling[230990]: clientsession.go:967: Subscribing video from Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== as 4 in session s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== Jul 14 20:40:03 turn signaling[230990]: mcu_janus.go:558: Started listener &{{janus.plugin.videoroom map[room:7752095335139882 started:ok videoroom:event]} map[] 1119253239952861 2171577364885873} Jul 14 20:40:03 turn signaling[230990]: mcu_janus.go:558: Started listener &{{janus.plugin.videoroom map[room:5938377545275208 started:ok videoroom:event]} map[] 1119253239952861 1602128575696101} Jul 14 20:40:03 turn signaling[230990]: mcu_janus.go:864: Publisher 4012031908138812 received connected Jul 14 20:40:04 turn signaling[230990]: mcu_janus.go:1107: Subscriber 2171577364885873 received connected Jul 14 20:40:04 turn signaling[230990]: mcu_janus.go:1107: Subscriber 1602128575696101 received connected Jul 14 20:40:24 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 9 ms (9.974681ms) Jul 14 20:40:53 turn signaling[230990]: client.go:282: Client Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== has RTT of 5 ms (5.744767ms) Jul 14 20:41:18 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 5 ms (5.047643ms) Jul 14 20:41:39 turn signaling[230990]: client.go:284: Client from 192.168.1.43 has RTT of 21 ms (21.430432ms) Jul 14 20:41:39 turn signaling[230990]: client.go:284: Client from 192.168.1.43 has RTT of 62 ms (62.949296ms) Jul 14 20:41:39 turn signaling[230990]: hub.go:740: Register user **** from 192.168.1.43 in unknown-country (Mozilla/5.0 (Android) Nextcloud-Talk v3177) X1fR4NzSUKyHxu8OkcRWzwXgXJcKEjC1zU_YGycQKsB8PT1BblBuR1d5cW9xWjNtMTRFZ3pkaUlsTVRXMEFMUjl0aUVDR0JaZUZMc2ZyYWphVzlWTWxBNHl5RFh6UWRFN3BzUVRPQkc4QlMwNVZ2QU9jN2tGSlZ6Z2JfM2FtLXZwcWVzMkIxYVU1Y1h3UjZxWi0wd2RRWTRuMVh1YUFHTy1sY1dTYjhFSkRKV2JyLV82b2hfOUU3S3hMLVlWbTQ1ak1ISlhvQkxqYVc4M1ZNQzJ8OTk2MzYzOTg2MQ== (private=MTY4OTM2MzY5OXw5N1ZoZGxXbG1pRmRSc1NWNHQxY1psV09Xb0tNU04yNDNRYjUzbkFjc25MRG5VWlk4bkFUdHhNbFJXNU1TeXN3a19uYm1hVV9BNVZpVDhxMmJNNmxwajZhaDdVc2RKaVVOa1p1UkV3VDY4Zkd3ZUhqYnl6MVlJU01NWU4xVWlLaG1HakJZSmx4LWtaYldDam02N2Q3U1JTcWdxeVZib2xkeVdYTXZic1N5VldOM2c9PXx4LnVxF0NCZmr4iP7Pf05mBYxxqnJpqk-HTUAx04WFUw==) **Jul 14 20:41:41 turn signaling[230990]: client.go:303: Error reading from 192.168.1.43: read tcp 127.0.0.1:8080->127.0.0.1:37716: use of closed network connection** Jul 14 20:41:47 turn signaling[230990]: client.go:282: Client Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== has RTT of 4 ms (4.323438ms) Jul 14 20:41:49 turn signaling[230990]: client.go:301: Error reading from client X1fR4NzSUKyHxu8OkcRWzwXgXJcKEjC1zU_YGycQKsB8PT1BblBuR1d5cW9xWjNtMTRFZ3pkaUlsTVRXMEFMUjl0aUVDR0JaZUZMc2ZyYWphVzlWTWxBNHl5RFh6UWRFN3BzUVRPQkc4QlMwNVZ2QU9jN2tGSlZ6Z2JfM2FtLXZwcWVzMkIxYVU1Y1h3UjZxWi0wd2RRWTRuMVh1YUFHTy1sY1dTYjhFSkRKV2JyLV82b2hfOUU3S3hMLVlWbTQ1ak1ISlhvQkxqYVc4M1ZNQzJ8OTk2MzYzOTg2MQ==: websocket: close 1006 (abnormal closure): unexpected EOF Jul 14 20:41:49 turn signaling[230990]: hub.go:832: Unregister X1fR4NzSUKyHxu8OkcRWzwXgXJcKEjC1zU_YGycQKsB8PT1BblBuR1d5cW9xWjNtMTRFZ3pkaUlsTVRXMEFMUjl0aUVDR0JaZUZMc2ZyYWphVzlWTWxBNHl5RFh6UWRFN3BzUVRPQkc4QlMwNVZ2QU9jN2tGSlZ6Z2JfM2FtLXZwcWVzMkIxYVU1Y1h3UjZxWi0wd2RRWTRuMVh1YUFHTy1sY1dTYjhFSkRKV2JyLV82b2hfOUU3S3hMLVlWbTQ1ak1ISlhvQkxqYVc4M1ZNQzJ8OTk2MzYzOTg2MQ== (private=MTY4OTM2MzY5OXw5N1ZoZGxXbG1pRmRSc1NWNHQxY1psV09Xb0tNU04yNDNRYjUzbkFjc25MRG5VWlk4bkFUdHhNbFJXNU1TeXN3a19uYm1hVV9BNVZpVDhxMmJNNmxwajZhaDdVc2RKaVVOa1p1UkV3VDY4Zkd3ZUhqYnl6MVlJU01NWU4xVWlLaG1HakJZSmx4LWtaYldDam02N2Q3U1JTcWdxeVZib2xkeVdYTXZic1N5VldOM2c9PXx4LnVxF0NCZmr4iP7Pf05mBYxxqnJpqk-HTUAx04WFUw==) Jul 14 20:41:58 turn signaling[230990]: mcu_janus.go:1097: Subscriber 2171577364885873 received hangup (Close PC), closing Jul 14 20:41:58 turn signaling[230990]: mcu_janus.go:854: Publisher 4982243268801941 received hangup (DTLS alert), closing Jul 14 20:41:58 turn signaling[230990]: mcu_janus.go:841: Publisher 4982243268801941: associated room has been destroyed, closing Jul 14 20:41:58 turn signaling[230990]: mcu_janus.go:922: Room 7752095335139882 destroyed Jul 14 20:41:58 turn signaling[230990]: janus_client.go:470: Unable to deliver message { Jul 14 20:41:58 turn signaling[230990]: "janus": "detached", Jul 14 20:41:58 turn signaling[230990]: "session_id": 1119253239952861, Jul 14 20:41:58 turn signaling[230990]: "sender": 2171577364885873 Jul 14 20:41:58 turn signaling[230990]: }. Handle 2171577364885873 gone? Jul 14 20:41:58 turn signaling[230990]: janus_client.go:470: Unable to deliver message { Jul 14 20:41:58 turn signaling[230990]: "janus": "detached", Jul 14 20:41:58 turn signaling[230990]: "session_id": 1119253239952861, Jul 14 20:41:58 turn signaling[230990]: "sender": 4982243268801941 Jul 14 20:41:58 turn signaling[230990]: }. Handle 4982243268801941 gone? Jul 14 20:41:58 turn signaling[230990]: mcu_janus.go:1097: Subscriber 1602128575696101 received hangup (DTLS alert), closing Jul 14 20:41:58 turn signaling[230990]: janus_client.go:470: Unable to deliver message { Jul 14 20:41:58 turn signaling[230990]: "janus": "detached", Jul 14 20:41:58 turn signaling[230990]: "session_id": 1119253239952861, Jul 14 20:41:58 turn signaling[230990]: "sender": 1602128575696101 Jul 14 20:41:58 turn signaling[230990]: }. Handle 1602128575696101 gone? Jul 14 20:41:58 turn signaling[230990]: clientsession.go:448: Session s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== left call 77j45iw7 Jul 14 20:42:00 turn signaling[230990]: mcu_janus.go:854: Publisher 4012031908138812 received hangup (DTLS alert), closing Jul 14 20:42:00 turn signaling[230990]: mcu_janus.go:922: Room 5938377545275208 destroyed Jul 14 20:42:00 turn signaling[230990]: janus_client.go:470: Unable to deliver message { Jul 14 20:42:00 turn signaling[230990]: "janus": "detached", Jul 14 20:42:00 turn signaling[230990]: "session_id": 1119253239952861, Jul 14 20:42:00 turn signaling[230990]: "sender": 4012031908138812 Jul 14 20:42:00 turn signaling[230990]: }. Handle 4012031908138812 gone? Jul 14 20:42:00 turn signaling[230990]: clientsession.go:448: Session Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== left call 77j45iw7 Jul 14 20:42:12 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 9 ms (9.713171ms) Jul 14 20:42:14 turn signaling[230990]: room.go:656: Session s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== joined call 77j45iw7 Jul 14 20:42:14 turn signaling[230990]: mcu_janus.go:729: Attached video as publisher 7438568151038063 to plugin janus.plugin.videoroom in session 1119253239952861 Jul 14 20:42:14 turn signaling[230990]: mcu_janus.go:767: Created room 8067434077290886 {janus.plugin.videoroom map[permanent:false room:8067434077290886 videoroom:created]} Jul 14 20:42:14 turn signaling[230990]: mcu_janus.go:825: Publisher s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== is using handle 7438568151038063 Jul 14 20:42:14 turn signaling[230990]: clientsession.go:890: Publishing video as 5 for session s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== Jul 14 20:42:14 turn signaling[230990]: mcu_janus.go:864: Publisher 7438568151038063 received connected Jul 14 20:42:18 turn signaling[230990]: client.go:301: Error reading from client Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ==: read tcp 127.0.0.1:8080->127.0.0.1:60714: use of closed network connection Jul 14 20:42:18 turn signaling[230990]: hub.go:832: Unregister Wppg-EqelE5EBuj-L0fqkak-QBCWf8q77GqHT4L6nYl8PT13YTFKdmFVTUtxc3VPVE5mVlJ1RkFyZnE4dHpwQlJNQldBM1Q1YllkSUhIaFlUQ0dwTnhWeldMaC1Gb3ZJd1NTekZOalkxVWlRNmZvY2ZfQ3ZNODFVWGxMdWp1T2V0RDI1aEZuUElzTktGUzRqVkdwZ0Z2RUxSRFFMdGVxYmNpYWs3WUdlNVlDbjc2TjFTZ0VrYXQwaW1pWlQ3azNRWnhMTXhRMnpHSTZlVUVBSy18OTk1MzYzOTg2MQ== (private=MTY4OTM2MzU5OXxUTEZSbUxRRDZtSXdxRHZBR21Nb3VhYWtWb3ZiUTRxZTFneHFudW0yT3ZDMXZzNlF0SUFUOW9sMVZsSUV1ek9iaUU4ejFHRmVfVUtkV0xkUjdhNVlUem1icGRIVmpVS1BJcGMyVmJvYUlwZ283b2RPcVFkUkxkWnlXU09qUUE3VVRNNzVnZUY4YlNUQ1lhRG0zX3VrRFF5b1hsNjMxN0Q2N0pHNFR2dW1KaW51NXc9PXxXlFnlMRXh0_0qwDo_nV6zJsgsOPR-NhngLg4vW-R5MQ==) Jul 14 20:42:18 turn signaling[230990]: clientsession.go:491: Removed room session /AEiw2Bp+w7H8jjX78lCNU+3jVGOJF+u7TH3YLEfIN3Fs2m2HxGy3cDD8IsDI410Pk2Yu6D1rGeRhwlP3Dm8djtD4xc1vurRL8YgkC9NRNARI9Y5sztXiUPOHa8KEPxEMyVffixJhhXgHVS52hs6DaBsk8Dyw28/c6vbUM9JO7vUCt0miMR0WX6GQcK/noRUy8QE2T1MqoYakeEUXY1tQAYQuJ6UoE1/ukSWlYNs0E5B+YpNnYno0U+hqGi7cfW: map[error:map[code:no_such_room message:The user is not invited to this room.] type:error] Jul 14 20:42:19 turn signaling[230990]: hub.go:585: Closing expired session X1fR4NzSUKyHxu8OkcRWzwXgXJcKEjC1zU_YGycQKsB8PT1BblBuR1d5cW9xWjNtMTRFZ3pkaUlsTVRXMEFMUjl0aUVDR0JaZUZMc2ZyYWphVzlWTWxBNHl5RFh6UWRFN3BzUVRPQkc4QlMwNVZ2QU9jN2tGSlZ6Z2JfM2FtLXZwcWVzMkIxYVU1Y1h3UjZxWi0wd2RRWTRuMVh1YUFHTy1sY1dTYjhFSkRKV2JyLV82b2hfOUU3S3hMLVlWbTQ1ak1ISlhvQkxqYVc4M1ZNQzJ8OTk2MzYzOTg2MQ== (private=MTY4OTM2MzY5OXw5N1ZoZGxXbG1pRmRSc1NWNHQxY1psV09Xb0tNU04yNDNRYjUzbkFjc25MRG5VWlk4bkFUdHhNbFJXNU1TeXN3a19uYm1hVV9BNVZpVDhxMmJNNmxwajZhaDdVc2RKaVVOa1p1UkV3VDY4Zkd3ZUhqYnl6MVlJU01NWU4xVWlLaG1HakJZSmx4LWtaYldDam02N2Q3U1JTcWdxeVZib2xkeVdYTXZic1N5VldOM2c9PXx4LnVxF0NCZmr4iP7Pf05mBYxxqnJpqk-HTUAx04WFUw==) Jul 14 20:42:45 turn signaling[230990]: mcu_janus.go:854: Publisher 7438568151038063 received hangup (DTLS alert), closing Jul 14 20:42:45 turn signaling[230990]: mcu_janus.go:841: Publisher 7438568151038063: associated room has been destroyed, closing Jul 14 20:42:45 turn signaling[230990]: mcu_janus.go:922: Room 8067434077290886 destroyed Jul 14 20:42:45 turn signaling[230990]: janus_client.go:470: Unable to deliver message { Jul 14 20:42:45 turn signaling[230990]: "janus": "detached", Jul 14 20:42:45 turn signaling[230990]: "session_id": 1119253239952861, Jul 14 20:42:45 turn signaling[230990]: "sender": 7438568151038063 Jul 14 20:42:45 turn signaling[230990]: }. Handle 7438568151038063 gone? Jul 14 20:42:45 turn signaling[230990]: clientsession.go:448: Session s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== left call 77j45iw7 Jul 14 20:43:06 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 96 ms (96.688074ms) Jul 14 20:44:00 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 59 ms (59.394776ms) Jul 14 20:44:54 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 125 ms (125.677257ms) Jul 14 20:45:48 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 90 ms (90.198363ms) Jul 14 20:46:42 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 52 ms (52.744835ms) Jul 14 20:47:36 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 120 ms (120.339811ms) Jul 14 20:47:47 turn signaling[230990]: clientsession.go:491: Removed room session pTjb3prXdrJk4DgnuWUMrQR4o9hPbmFJ7MsvWeNesuaN7zihRJoKuu69ymqtnh0LTCxHnzuGsbx1HgfvCV2zY6fNmoMrExDdh3ulF/gVS7yfByPqZXq2X43eee45zMIkzs/lxKa3z1ZCfW4g1lvxvn+cxOGbYOrsUM8tg9PPhMuQs2ZJRo80m255rTXcKgfBcP18vvdHcr6E93hYSjX0MNqt/qlxG2mRIcrUEsZ9yrkBegLeXwaDmApOwaohyb1: map[error:map[code:no_such_room message:The user is not invited to this room.] type:error] Jul 14 20:48:30 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 84 ms (84.895903ms) Jul 14 20:49:24 turn signaling[230990]: client.go:282: Client s7uyHpnDPrjgTrF7tzPK5N9GGYfLZmw8_er30f0MziF8PT1nUEduWGJQTHhnd0xoV09jSFp6MGR5YVJvbzRORHhka3Y3Y3VjUzFqQ19QOWV5SVZBeWRuekxyTXpISmxTY3BXa0J0RGwwUGFsR2lDbTA2VC1pZF90SUVrcTlDcmRiRUpVbUhZbHpJUEwxRi1wRFNPZTY5d2RFWVd1MDNSeDF0ZlVVNk41Y19LT0dncWplVVVEX1RLX0EzS2J6ZzNaNW1yUmp2NkJJdWZDSUNZQmV8MDc1MzYzOTg2MQ== has RTT of 48 ms (48.282366ms)
</details>
attempted to highlight error details error use of closed network connection - N.B both clients behind LAN often when using mobile data it shows error

as https://github.com/strukturag/nextcloud-spreed-signaling/issues/511 the system keeper has kindly pointed out - much much appreciative of the direction over there i still have the issue after using 3177

 i only have the QA app open: v3177 but it seems to register the 17.0.1 and get the weird stuff happening

<details>
<summary>Signalling Log </summary>
<br>
Jul 15 09:03:11 turn signaling[866]: mcu_janus.go:294: Connected to Janus WebRTC Server 0.14.0 by Meetecho s.r.l.
Jul 15 09:03:11 turn signaling[866]: mcu_janus.go:300: Found JANUS VideoRoom plugin 0.0.9 by Meetecho s.r.l.
Jul 15 09:03:11 turn signaling[866]: mcu_janus.go:305: Data channels are supported
Jul 15 09:03:11 turn signaling[866]: mcu_janus.go:309: Full-Trickle is enabled
Jul 15 09:03:11 turn signaling[866]: mcu_janus.go:311: Maximum bandwidth 1048576 bits/sec per publishing stream
Jul 15 09:03:11 turn signaling[866]: mcu_janus.go:312: Maximum bandwidth 2097152 bits/sec per screensharing stream
Jul 15 09:03:11 turn signaling[866]: mcu_janus.go:318: Created Janus session 996662133067120
Jul 15 09:03:11 turn signaling[866]: mcu_janus.go:325: Created Janus handle 1488034437752407
Jul 15 09:03:11 turn signaling[866]: main.go:263: Using janus MCU
Jul 15 09:03:11 turn signaling[866]: hub.go:385: Using a timeout of 10s for MCU requests
Jul 15 09:03:11 turn signaling[866]: backend_server.go:95: Using configured TURN API key
Jul 15 09:03:11 turn signaling[866]: backend_server.go:96: Using configured shared TURN secret
Jul 15 09:03:11 turn signaling[866]: backend_server.go:98: Adding "turn:127.0.0.1:3478?transport=udp" as TURN server
Jul 15 09:03:11 turn signaling[866]: backend_server.go:98: Adding "turn:127.0.0.1:3478?transport=tcp" as TURN server
Jul 15 09:03:11 turn signaling[866]: backend_server.go:111: No IPs configured for the stats endpoint, only allowing access from 127.0.0.1
Jul 15 09:03:11 turn signaling[866]: main.go:339: Listening on 127.0.0.1:8080
Jul 15 09:03:44 turn signaling[866]: client.go:284: Client from 192.168.1.203 has RTT of 9 ms (9.578461ms)
Jul 15 09:03:44 turn signaling[866]: client.go:284: Client from 192.168.1.203 has RTT of 14 ms (14.806824ms)
Jul 15 09:03:44 turn signaling[866]: capabilities.go:151: Capabilities expired for https://cloud.wh-computers.co.uk/ocs/v2.php/cloud/capabilities, updating
Jul 15 09:03:44 turn signaling[866]: client.go:284: Client from 192.168.1.203 has RTT of 53 ms (53.046038ms)
Jul 15 09:03:44 turn signaling[866]: capabilities.go:151: Capabilities expired for https://cloud.wh-computers.co.uk/ocs/v2.php/cloud/capabilities, updating
Jul 15 09:03:44 turn signaling[866]: capabilities.go:214: Received capabilities map[config:map[attachments:map[allowed:false] call:map[breakout-rooms:true can-upload-background:false enabled:true predefined-backgrounds:[1_office.jpg 2_home.jpg 3_abstract.jpg 4_beach.jpg 5_park.jpg 6_theater.jpg 7_library.jpg 8_space_station.jpg] recording:false supported-reactions:[❤️ 🎉 👏 👍 👎 😂 🤩 🤔 😲 😥]] chat:map[max-length:32000 read-privacy:0 translations:[] typing-privacy:0] conversations:map[can-create:false] previews:map[max-gif-size:3.145728e+06] signaling:map[hello-v2-token-key:-----BEGIN PUBLIC KEY-----
Jul 15 09:03:44 turn signaling[866]: MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEVKD0oL/yvpEzIxJAsKKRwzGUKkvo
Jul 15 09:03:44 turn signaling[866]: nLc7qH9x4xKtAeJj14x3XplFBXlFU1l5WZBEPvicCoTrLa/guCZX8wGxdg==
Jul 15 09:03:44 turn signaling[866]: -----END PUBLIC KEY-----
Jul 15 09:03:44 turn signaling[866]:  session-ping-limit:200]] features:[audio video chat-v2 conversation-v4 guest-signaling empty-group-room guest-display-names multi-room-users favorites last-room-activity no-ping system-messages delete-messages mention-flag in-call-flags conversation-call-flags notification-levels invite-groups-and-mails locked-one-to-one-rooms read-only-rooms listable-rooms chat-read-marker chat-unread webinary-lobby start-call-flag chat-replies circles-support force-mute sip-support sip-support-nopin chat-read-status phonebook-search raise-hand room-description rich-object-sharing temp-user-avatar-api geo-location-sharing voice-message-sharing signaling-v3 publishing-permissions clear-history direct-mention-flag notification-calls conversation-permissions rich-object-list-media rich-object-delete unified-search chat-permission silent-send silent-call send-call-notification talk-polls breakout-rooms-v1 recording-v1 avatar chat-get-context single-conversation-status chat-keep-notifications typing-privacy message-expiration reactions chat-reference-id] version:17.0.1] from https://cloud.wh-computers.co.uk/ocs/v2.php/cloud/capabilities
Jul 15 09:03:44 turn signaling[866]: capabilities.go:214: Received capabilities map[config:map[attachments:map[allowed:false] call:map[breakout-rooms:true can-upload-background:false enabled:true predefined-backgrounds:[1_office.jpg 2_home.jpg 3_abstract.jpg 4_beach.jpg 5_park.jpg 6_theater.jpg 7_library.jpg 8_space_station.jpg] recording:false supported-reactions:[❤️ 🎉 👏 👍 👎 😂 🤩 🤔 😲 😥]] chat:map[max-length:32000 read-privacy:0 translations:[] typing-privacy:0] conversations:map[can-create:false] previews:map[max-gif-size:3.145728e+06] signaling:map[hello-v2-token-key:-----BEGIN PUBLIC KEY-----
Jul 15 09:03:44 turn signaling[866]: MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEVKD0oL/yvpEzIxJAsKKRwzGUKkvo
Jul 15 09:03:44 turn signaling[866]: nLc7qH9x4xKtAeJj14x3XplFBXlFU1l5WZBEPvicCoTrLa/guCZX8wGxdg==
Jul 15 09:03:44 turn signaling[866]: -----END PUBLIC KEY-----
Jul 15 09:03:44 turn signaling[866]:  session-ping-limit:200]] features:[audio video chat-v2 conversation-v4 guest-signaling empty-group-room guest-display-names multi-room-users favorites last-room-activity no-ping system-messages delete-messages mention-flag in-call-flags conversation-call-flags notification-levels invite-groups-and-mails locked-one-to-one-rooms read-only-rooms listable-rooms chat-read-marker chat-unread webinary-lobby start-call-flag chat-replies circles-support force-mute sip-support sip-support-nopin chat-read-status phonebook-search raise-hand room-description rich-object-sharing temp-user-avatar-api geo-location-sharing voice-message-sharing signaling-v3 publishing-permissions clear-history direct-mention-flag notification-calls conversation-permissions rich-object-list-media rich-object-delete unified-search chat-permission silent-send silent-call send-call-notification talk-polls breakout-rooms-v1 recording-v1 avatar chat-get-context single-conversation-status chat-keep-notifications typing-privacy message-expiration reactions chat-reference-id] version:17.0.1] from https://cloud.wh-computers.co.uk/ocs/v2.php/cloud/capabilities
Jul 15 09:03:44 turn signaling[866]: client.go:284: Client from 192.168.1.203 has RTT of 58 ms (58.23642ms)
Jul 15 09:03:44 turn signaling[866]: hub.go:740: Register user ****@backend-1 from 192.168.1.203 in unknown-country (Mozilla/5.0 (Android) Nextcloud-Talk v17.0.1) H1VWqkJu5NHBMi9v9rmH85vJy4JjO3AjDKhj247wY3d8PT1BTmE2UUJKRzZXRUhGVi1nbmhMc0p4VEdWUFpVUjEzb3FRRUQ1X1huNHl0LXZfRS13bC16VzBVdzBsbU1uSkhJSVlLUVBRbmZKRkh2R2FCTTY2UFM3WkxBOFp0anhpODJqLVI4OWJwNE1HNF9RaHpROWE2WWMxbjVPbFdvQ0RKSm1VSU1PNVl6V3NtQkYteU5SaDQzeVlGN3FHOWY5LXFlMFZia2p2dG5sS25qQ1B8NDIyODA0OTg2MQ== (private=MTY4OTQwODIyNHx2bG9ackJnMVl2Sk9Mbk54MXM4U0Q5aGdPaXZ4OUFzOGZqWE9YdnREYmlaTFBRX3U4eHN0NlU5OFlhWkw2by11MGhvUmptU3hSQ0NWc0dRMmp0RXRQbDZsdjJoUGlBVGxnUUpNWHRmMTZpUjN5TUoxRUlaVlgwWDl2bnp6QVlRb0FnR0p3LUlCSWpSeTRUVkloOGhyczRfZDNuZDFVM3VnNWNxajREMDhqcW80d1E9PXxvSl2ZAmoGnQ593MJK5fGZPM4_Q8tn_gJzp_Ip6dwCWw==)
Jul 15 09:03:44 turn signaling[866]: hub.go:740: Register user ****@backend-1 from 192.168.1.203 in unknown-country (Mozilla/5.0 (Android) Nextcloud-Talk v3177) 7L85IZHgrgdCYwYHStevhTGcdpXaMZhlRnSjYJ7aHsx8PT1BYVRLUmlGY1ZtaDhCdXRxcG5nQkNkVGRlN1ZKRFpOaFBYWWY3Yzlac1hUU28zT3dRcUM4UjRpVURQbjBBV2daM3JmS0JtYkNxQ05VdWJLVG5FVHcxZ0IzdGJmNGN0M0tneV9rd25KU2R2UG9yMGJKdEVOUnBkS1hfSlNhNjNicmtFamFZODZTMTNTNVVieVIzZUtHLTNXUU5nSXJlT3FONkRLSG41SkwzaDhpZVh8NDIyODA0OTg2MQ== (private=MTY4OTQwODIyNHxheGZzWXZHTmFrVUVVbnlVNURONl9Cd3l4Zk5kREd0blJBZXZYNkFSNTluQWJLRWdhd3JnM2tnR0NHMG1mZzRMbXp5LThtREFqemd2b016M09fSEhsY2xkUjRleGZfdGI5aXV3ajFLclU4LUlGZmNhX21QOERUeWVqb2dZVjJubGZZMm9sWk1RVWFfYnl1a3VsWTl5ZHRUelAyVHlieXhZSW5HYi15NnhGeTVwMWc9PXwdAmTP9nD7qb3Bn_UxweOSStFxSkfgvdcnjWRz7cL4rw==)
Jul 15 09:03:44 turn signaling[866]: clientsession.go:434: Session 7L85IZHgrgdCYwYHStevhTGcdpXaMZhlRnSjYJ7aHsx8PT1BYVRLUmlGY1ZtaDhCdXRxcG5nQkNkVGRlN1ZKRFpOaFBYWWY3Yzlac1hUU28zT3dRcUM4UjRpVURQbjBBV2daM3JmS0JtYkNxQ05VdWJLVG5FVHcxZ0IzdGJmNGN0M0tneV9rd25KU2R2UG9yMGJKdEVOUnBkS1hfSlNhNjNicmtFamFZODZTMTNTNVVieVIzZUtHLTNXUU5nSXJlT3FONkRLSG41SkwzaDhpZVh8NDIyODA0OTg2MQ== joined room 77j45iw7 with room session id 9nZnMJteQxzPV3yJ67LXPq5zftIdbZkxbtt44fB6DXj9N46J4ugvOfZRKKREu8GOASJkJ4nhFJRWfwrunQOz+Ohj9f4Ran9RJ7Vt74BVc4av//Iw3QtWv3ynTArms/axYYoWIi3ZIQ1pk0E4lYoP64buHCnM0nej/OYIFaypOkVhQD/Hzg4PtnQYtLjtNPJc9FWfboardRqZHXRmKKdPhOhLBThII+FN2ZvEX6AYi6sZ+8ef6HBJP4pXwS/jVGs
Jul 15 09:03:44 turn signaling[866]: clientsession.go:286: Permissions of session 7L85IZHgrgdCYwYHStevhTGcdpXaMZhlRnSjYJ7aHsx8PT1BYVRLUmlGY1ZtaDhCdXRxcG5nQkNkVGRlN1ZKRFpOaFBYWWY3Yzlac1hUU28zT3dRcUM4UjRpVURQbjBBV2daM3JmS0JtYkNxQ05VdWJLVG5FVHcxZ0IzdGJmNGN0M0tneV9rd25KU2R2UG9yMGJKdEVOUnBkS1hfSlNhNjNicmtFamFZODZTMTNTNVVieVIzZUtHLTNXUU5nSXJlT3FONkRLSG41SkwzaDhpZVh8NDIyODA0OTg2MQ== changed: [publish-audio publish-video publish-screen control]
**Jul 15 09:03:47 turn signaling[866]: client.go:303: Error reading from 192.168.1.203: read tcp 127.0.0.1:8080->127.0.0.1:52126: use of closed network connection
Jul 15 09:03:47 turn signaling[866]: client.go:303: Error reading from 192.168.1.203: read tcp 127.0.0.1:8080->127.0.0.1:52114: use of closed network connection**
Jul 15 09:04:05 turn signaling[866]: clientsession.go:491: Removed room session 9nZnMJteQxzPV3yJ67LXPq5zftIdbZkxbtt44fB6DXj9N46J4ugvOfZRKKREu8GOASJkJ4nhFJRWfwrunQOz+Ohj9f4Ran9RJ7Vt74BVc4av//Iw3QtWv3ynTArms/axYYoWIi3ZIQ1pk0E4lYoP64buHCnM0nej/OYIFaypOkVhQD/Hzg4PtnQYtLjtNPJc9FWfboardRqZHXRmKKdPhOhLBThII+FN2ZvEX6AYi6sZ+8ef6HBJP4pXwS/jVGs: map[error:map[code:no_such_room message:The user is not invited to this room.] type:error]
Jul 15 09:04:06 turn signaling[866]: clientsession.go:434: Session 7L85IZHgrgdCYwYHStevhTGcdpXaMZhlRnSjYJ7aHsx8PT1BYVRLUmlGY1ZtaDhCdXRxcG5nQkNkVGRlN1ZKRFpOaFBYWWY3Yzlac1hUU28zT3dRcUM4UjRpVURQbjBBV2daM3JmS0JtYkNxQ05VdWJLVG5FVHcxZ0IzdGJmNGN0M0tneV9rd25KU2R2UG9yMGJKdEVOUnBkS1hfSlNhNjNicmtFamFZODZTMTNTNVVieVIzZUtHLTNXUU5nSXJlT3FONkRLSG41SkwzaDhpZVh8NDIyODA0OTg2MQ== joined room 77j45iw7 with room session id c+1MAH8g1j3dJNF0GtGw9Go9biK9dl/BpMISP38b2OW04z8PZz7cYEVULBBfBsR5lJFEKv9nAASmYCEAuDUhMKcP7XQovbIBR04zdkwHblMCoMrv8qzwqLCCyj6anE8c5dtAjJ1vGMy/hauEyEMTAOI2Q85Ig9MuzXLCe9FRDSuH65eogwDxJX9SWxW2dNLWXJYLTu/19LlJXc98/+Ig52lmhpgxhXD4jXpdHhN5jJ/+xp95M9u+A8I9TnKofk5
</details>

i only have the QA app open but it registers 17.0.1??? 

2nd check 
<details>
<summary>Signalling Log 2 </summary>
<br>
Jul 15 09:12:06 turn systemd[1]: Started Nextcloud Talk signaling server.
Jul 15 09:12:06 turn signaling[2430]: main.go:133: Starting up version 12de5a9b71afef06eaf171099712ef0a5f6ca0c4/go1.20.5 as pid 2430
Jul 15 09:12:06 turn signaling[2430]: main.go:142: Using a maximum of 2 CPUs
Jul 15 09:12:06 turn signaling[2430]: natsclient.go:108: Connection established to nats://127.0.0.1:4222 (NB54B2CZ7DDYRTZYNL4SGJG2Q3OMKCJMZHPWVVZUJIBQVHNQGMCT7RQX)
Jul 15 09:12:06 turn signaling[2430]: grpc_common.go:167: WARNING: No GRPC server certificate and/or key configured, running unencrypted
Jul 15 09:12:06 turn signaling[2430]: grpc_common.go:169: WARNING: No GRPC CA configured, expecting unencrypted connections
Jul 15 09:12:06 turn signaling[2430]: backend_storage_static.go:72: Backend backend-1 added for https://cloud.wh-computers.co.uk/
Jul 15 09:12:06 turn signaling[2430]: hub.go:200: Using a maximum of 8 concurrent backend connections per host
Jul 15 09:12:06 turn signaling[2430]: hub.go:207: Using a timeout of 10s for backend connections
Jul 15 09:12:06 turn signaling[2430]: hub.go:303: Not using GeoIP database
Jul 15 09:12:06 turn signaling[2430]: mcu_janus.go:294: Connected to Janus WebRTC Server 0.14.0 by Meetecho s.r.l.
Jul 15 09:12:06 turn signaling[2430]: mcu_janus.go:300: Found JANUS VideoRoom plugin 0.0.9 by Meetecho s.r.l.
Jul 15 09:12:06 turn signaling[2430]: mcu_janus.go:305: Data channels are supported
Jul 15 09:12:06 turn signaling[2430]: mcu_janus.go:309: Full-Trickle is enabled
Jul 15 09:12:06 turn signaling[2430]: mcu_janus.go:311: Maximum bandwidth 1048576 bits/sec per publishing stream
Jul 15 09:12:06 turn signaling[2430]: mcu_janus.go:312: Maximum bandwidth 2097152 bits/sec per screensharing stream
Jul 15 09:12:06 turn signaling[2430]: mcu_janus.go:318: Created Janus session 4177674057690730
Jul 15 09:12:06 turn signaling[2430]: mcu_janus.go:325: Created Janus handle 5478046910507472
Jul 15 09:12:06 turn signaling[2430]: main.go:263: Using janus MCU
Jul 15 09:12:06 turn signaling[2430]: hub.go:385: Using a timeout of 10s for MCU requests
Jul 15 09:12:06 turn signaling[2430]: backend_server.go:95: Using configured TURN API key
Jul 15 09:12:06 turn signaling[2430]: backend_server.go:96: Using configured shared TURN secret
Jul 15 09:12:06 turn signaling[2430]: backend_server.go:98: Adding "turn:127.0.0.1:3478?transport=udp" as TURN server
Jul 15 09:12:06 turn signaling[2430]: backend_server.go:98: Adding "turn:127.0.0.1:3478?transport=tcp" as TURN server
Jul 15 09:12:06 turn signaling[2430]: backend_server.go:111: No IPs configured for the stats endpoint, only allowing access from 127.0.0.1
Jul 15 09:12:06 turn signaling[2430]: main.go:339: Listening on 127.0.0.1:8080
Jul 15 09:12:27 turn signaling[2430]: client.go:284: Client from 192.168.1.203 has RTT of 9 ms (9.85069ms)
Jul 15 09:12:27 turn signaling[2430]: client.go:284: Client from 192.168.1.203 has RTT of 13 ms (13.343974ms)
Jul 15 09:12:27 turn signaling[2430]: capabilities.go:151: Capabilities expired for https://cloud.wh-computers.co.uk/ocs/v2.php/cloud/capabilities, updating
Jul 15 09:12:27 turn signaling[2430]: client.go:284: Client from 192.168.1.203 has RTT of 49 ms (49.359953ms)
Jul 15 09:12:27 turn signaling[2430]: capabilities.go:214: Received capabilities map[config:map[attachments:map[allowed:false] call:map[breakout-rooms:true can-upload-background:false enabled:true predefined-backgrounds:[1_office.jpg 2_home.jpg 3_abstract.jpg 4_beach.jpg 5_park.jpg 6_theater.jpg 7_library.jpg 8_space_station.jpg] recording:false supported-reactions:[❤️ 🎉 👏 👍 👎 😂 🤩 🤔 😲 😥]] chat:map[max-length:32000 read-privacy:0 translations:[] typing-privacy:0] conversations:map[can-create:false] previews:map[max-gif-size:3.145728e+06] signaling:map[hello-v2-token-key:-----BEGIN PUBLIC KEY-----
Jul 15 09:12:27 turn signaling[2430]: MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEVKD0oL/yvpEzIxJAsKKRwzGUKkvo
Jul 15 09:12:27 turn signaling[2430]: nLc7qH9x4xKtAeJj14x3XplFBXlFU1l5WZBEPvicCoTrLa/guCZX8wGxdg==
Jul 15 09:12:27 turn signaling[2430]: -----END PUBLIC KEY-----
Jul 15 09:12:27 turn signaling[2430]:  session-ping-limit:200]] features:[audio video chat-v2 conversation-v4 guest-signaling empty-group-room guest-display-names multi-room-users favorites last-room-activity no-ping system-messages delete-messages mention-flag in-call-flags conversation-call-flags notification-levels invite-groups-and-mails locked-one-to-one-rooms read-only-rooms listable-rooms chat-read-marker chat-unread webinary-lobby start-call-flag chat-replies circles-support force-mute sip-support sip-support-nopin chat-read-status phonebook-search raise-hand room-description rich-object-sharing temp-user-avatar-api geo-location-sharing voice-message-sharing signaling-v3 publishing-permissions clear-history direct-mention-flag notification-calls conversation-permissions rich-object-list-media rich-object-delete unified-search chat-permission silent-send silent-call send-call-notification talk-polls breakout-rooms-v1 recording-v1 avatar chat-get-context single-conversation-status chat-keep-notifications typing-privacy message-expiration reactions chat-reference-id] version:17.0.1] from https://cloud.wh-computers.co.uk/ocs/v2.php/cloud/capabilities
Jul 15 09:12:27 turn signaling[2430]: hub.go:740: Register user wayne@backend-1 from 192.168.1.203 in unknown-country (Mozilla/5.0 (Android) Nextcloud-Talk v17.0.1) aTX587IW_citz2tS2eNCYnAfwaQOp78nACqWAOJFPud8PT13TWQ5ZWpiRnZkRUthd3p4RUZmaDJkU08wNXhYOG9oeXJDekdpVTRvTDFsb0xEajZ3LXZHeGRxRXlFa0Rtc1NCdl9GV1FsZXVxQWRwME5ILTZrOV94b1FLdHBzUXpSYjZxSDd5bGpVd2p1S01oYWtPbXJnYlIzSlJCU1BWSWdBODFuU0JtYTYtUUxXaE53bFBVeUp0bzdmWGxwd015NElnYlNyZXVyYUFCU3JlSUd8NzQ3ODA0OTg2MQ== (private=MTY4OTQwODc0N3w1WDdUaHE0NGlwMlByQkZacldNRVZUeVhWMUpYdjZuWnZFeUpBbEppaS1fRFlJYUdNdW1odnJ6ME9XYXdpa2RKOG5Rajdpai1VcW03M1pyV1lvSkY0MXBDZW8zUjJCX1VaZ1J2ZzZwclI4RTA1WWJuY2FXaFVMUnQ4ZlFGd1QzR0ZiMmEzcmxUR1p1Ulp1N3BHcFFodGt2UDd0S1lFZ1B4TGRNUk5RbmJMbzN1SkE9PXzb5_GxRtxNXZs9I3gWy-V_6VZ7ROXPwo82Fhwonv8x_Q==)
Jul 15 09:12:27 turn signaling[2430]: hub.go:740: Register user wayne@backend-1 from 192.168.1.203 in unknown-country (Mozilla/5.0 (Android) Nextcloud-Talk v3177) IF_gNVm-TZW9RaZ2-nH8I4vaI9mUtFCJZ-IYMUGuQSZ8PT1RZnBrZjVOVWVWMUs4YkRnV09WRFIxM1BBZ245anBHY3ZaV3QwTWdVbTE0ZXo3ak1hZGE3dHZtSVMtcDl6bkZlTTdVWTV5dnpmV0NLbXBlLVFXNGtZVmtNbTZQWF9fOGppUXAzUTctOXdTQzNubHIxU0ZyWWdnaW1SYUZHM3NLaVRfOWJKODZEdzR0d1FSQ2RuaXZEanJsLU1iTHN0dUhEM0ZNN2o0eDNEejFTR258NzQ3ODA0OTg2MQ== (private=MTY4OTQwODc0N3x4eVlRWlhyX2hBSkdZc3o3STh5MWdkLWljd2NZd3VBOEtRd3JqNklrVXVUSl9SY2tJVWI4VTBPMWVRd1p6SFRaNUdWei05ckhiZTRmcjYyWHIyd2RwSG5VSEhXUHh6Rkh1cXFUanZJRnR6blpaQlRHVE9TNWxFUmVLclpKMl9PRjJwMFd0Zi1kLVpOQjIyUGNVY0pidXpJTG1BS0FqNUl2cFo2UlI2UFdfM2E5R2c9PXxzJ2xtd5GBvg2coYvE3zo8T-oZmto244z3vybk__WV1Q==)
Jul 15 09:12:27 turn signaling[2430]: client.go:282: Client IF_gNVm-TZW9RaZ2-nH8I4vaI9mUtFCJZ-IYMUGuQSZ8PT1RZnBrZjVOVWVWMUs4YkRnV09WRFIxM1BBZ245anBHY3ZaV3QwTWdVbTE0ZXo3ak1hZGE3dHZtSVMtcDl6bkZlTTdVWTV5dnpmV0NLbXBlLVFXNGtZVmtNbTZQWF9fOGppUXAzUTctOXdTQzNubHIxU0ZyWWdnaW1SYUZHM3NLaVRfOWJKODZEdzR0d1FSQ2RuaXZEanJsLU1iTHN0dUhEM0ZNN2o0eDNEejFTR258NzQ3ODA0OTg2MQ== has RTT of 43 ms (43.412083ms)
Jul 15 09:12:27 turn signaling[2430]: clientsession.go:434: Session IF_gNVm-TZW9RaZ2-nH8I4vaI9mUtFCJZ-IYMUGuQSZ8PT1RZnBrZjVOVWVWMUs4YkRnV09WRFIxM1BBZ245anBHY3ZaV3QwTWdVbTE0ZXo3ak1hZGE3dHZtSVMtcDl6bkZlTTdVWTV5dnpmV0NLbXBlLVFXNGtZVmtNbTZQWF9fOGppUXAzUTctOXdTQzNubHIxU0ZyWWdnaW1SYUZHM3NLaVRfOWJKODZEdzR0d1FSQ2RuaXZEanJsLU1iTHN0dUhEM0ZNN2o0eDNEejFTR258NzQ3ODA0OTg2MQ== joined room 77j45iw7 with room session id VwnJyU+t4y26i7b8XEduj5syGjGhCQeeCD8PIvN50b+KzkU97wiWm0zM6LGRxojxqKuloGaxVw79/OpnIiJPwZ7IUeJ9M9wLEOegN37xEZtJWiN5JbhyDbawVcIIxVH+l7kBswrKJWgKlKTcdVMZpeJRAumLHKwLfkN4Oern0wu4rPLegt5ZPG9OmUfLf9Dj97vfhkngxh74uR37ROuplQFiZNJPy4Mjk96aZzmfsIWKIapI48RyKDrba9lH7Bh
Jul 15 09:12:27 turn signaling[2430]: clientsession.go:286: Permissions of session IF_gNVm-TZW9RaZ2-nH8I4vaI9mUtFCJZ-IYMUGuQSZ8PT1RZnBrZjVOVWVWMUs4YkRnV09WRFIxM1BBZ245anBHY3ZaV3QwTWdVbTE0ZXo3ak1hZGE3dHZtSVMtcDl6bkZlTTdVWTV5dnpmV0NLbXBlLVFXNGtZVmtNbTZQWF9fOGppUXAzUTctOXdTQzNubHIxU0ZyWWdnaW1SYUZHM3NLaVRfOWJKODZEdzR0d1FSQ2RuaXZEanJsLU1iTHN0dUhEM0ZNN2o0eDNEejFTR258NzQ3ODA0OTg2MQ== changed: [publish-audio publish-video publish-screen control]
Jul 15 09:12:29 turn signaling[2430]: client.go:303: Error reading from 192.168.1.203: read tcp 127.0.0.1:8080->127.0.0.1:50102: use of closed network connection
Jul 15 09:12:29 turn signaling[2430]: client.go:303: Error reading from 192.168.1.203: read tcp 127.0.0.1:8080->127.0.0.1:50106: use of closed network connection
Jul 15 09:12:55 turn signaling[2430]: clientsession.go:491: Removed room session VwnJyU+t4y26i7b8XEduj5syGjGhCQeeCD8PIvN50b+KzkU97wiWm0zM6LGRxojxqKuloGaxVw79/OpnIiJPwZ7IUeJ9M9wLEOegN37xEZtJWiN5JbhyDbawVcIIxVH+l7kBswrKJWgKlKTcdVMZpeJRAumLHKwLfkN4Oern0wu4rPLegt5ZPG9OmUfLf9Dj97vfhkngxh74uR37ROuplQFiZNJPy4Mjk96aZzmfsIWKIapI48RyKDrba9lH7Bh: map[error:map[code:no_such_room message:The user is not invited to this room.] type:error]
Jul 15 09:13:22 turn signaling[2430]: client.go:282: Client aTX587IW_citz2tS2eNCYnAfwaQOp78nACqWAOJFPud8PT13TWQ5ZWpiRnZkRUthd3p4RUZmaDJkU08wNXhYOG9oeXJDekdpVTRvTDFsb0xEajZ3LXZHeGRxRXlFa0Rtc1NCdl9GV1FsZXVxQWRwME5ILTZrOV94b1FLdHBzUXpSYjZxSDd5bGpVd2p1S01oYWtPbXJnYlIzSlJCU1BWSWdBODFuU0JtYTYtUUxXaE53bFBVeUp0bzdmWGxwd015NElnYlNyZXVyYUFCU3JlSUd8NzQ3ODA0OTg2MQ== has RTT of 1049 ms (1.049354537s)
Jul 15 09:13:22 turn signaling[2430]: client.go:282: Client IF_gNVm-TZW9RaZ2-nH8I4vaI9mUtFCJZ-IYMUGuQSZ8PT1RZnBrZjVOVWVWMUs4YkRnV09WRFIxM1BBZ245anBHY3ZaV3QwTWdVbTE0ZXo3ak1hZGE3dHZtSVMtcDl6bkZlTTdVWTV5dnpmV0NLbXBlLVFXNGtZVmtNbTZQWF9fOGppUXAzUTctOXdTQzNubHIxU0ZyWWdnaW1SYUZHM3NLaVRfOWJKODZEdzR0d1FSQ2RuaXZEanJsLU1iTHN0dUhEM0ZNN2o0eDNEejFTR258NzQ3ODA0OTg2MQ== has RTT of 977 ms (977.890642ms)
Jul 15 09:14:15 turn signaling[2430]: client.go:282: Client aTX587IW_citz2tS2eNCYnAfwaQOp78nACqWAOJFPud8PT13TWQ5ZWpiRnZkRUthd3p4RUZmaDJkU08wNXhYOG9oeXJDekdpVTRvTDFsb0xEajZ3LXZHeGRxRXlFa0Rtc1NCdl9GV1FsZXVxQWRwME5ILTZrOV94b1FLdHBzUXpSYjZxSDd5bGpVd2p1S01oYWtPbXJnYlIzSlJCU1BWSWdBODFuU0JtYTYtUUxXaE53bFBVeUp0bzdmWGxwd015NElnYlNyZXVyYUFCU3JlSUd8NzQ3ODA0OTg2MQ== has RTT of 561 ms (561.517612ms)
Jul 15 09:14:15 turn signaling[2430]: client.go:282: Client IF_gNVm-TZW9RaZ2-nH8I4vaI9mUtFCJZ-IYMUGuQSZ8PT1RZnBrZjVOVWVWMUs4YkRnV09WRFIxM1BBZ245anBHY3ZaV3QwTWdVbTE0ZXo3ak1hZGE3dHZtSVMtcDl6bkZlTTdVWTV5dnpmV0NLbXBlLVFXNGtZVmtNbTZQWF9fOGppUXAzUTctOXdTQzNubHIxU0ZyWWdnaW1SYUZHM3NLaVRfOWJKODZEdzR0d1FSQ2RuaXZEanJsLU1iTHN0dUhEM0ZNN2o0eDNEejFTR258NzQ3ODA0OTg2MQ== has RTT of 487 ms (487.328883ms)
Jul 15 09:14:40 turn signaling[2430]: client.go:301: Error reading from client IF_gNVm-TZW9RaZ2-nH8I4vaI9mUtFCJZ-IYMUGuQSZ8PT1RZnBrZjVOVWVWMUs4YkRnV09WRFIxM1BBZ245anBHY3ZaV3QwTWdVbTE0ZXo3ak1hZGE3dHZtSVMtcDl6bkZlTTdVWTV5dnpmV0NLbXBlLVFXNGtZVmtNbTZQWF9fOGppUXAzUTctOXdTQzNubHIxU0ZyWWdnaW1SYUZHM3NLaVRfOWJKODZEdzR0d1FSQ2RuaXZEanJsLU1iTHN0dUhEM0ZNN2o0eDNEejFTR258NzQ3ODA0OTg2MQ==: websocket: close 1006 (abnormal closure): unexpected EOF
Jul 15 09:14:40 turn signaling[2430]: hub.go:832: Unregister IF_gNVm-TZW9RaZ2-nH8I4vaI9mUtFCJZ-IYMUGuQSZ8PT1RZnBrZjVOVWVWMUs4YkRnV09WRFIxM1BBZ245anBHY3ZaV3QwTWdVbTE0ZXo3ak1hZGE3dHZtSVMtcDl6bkZlTTdVWTV5dnpmV0NLbXBlLVFXNGtZVmtNbTZQWF9fOGppUXAzUTctOXdTQzNubHIxU0ZyWWdnaW1SYUZHM3NLaVRfOWJKODZEdzR0d1FSQ2RuaXZEanJsLU1iTHN0dUhEM0ZNN2o0eDNEejFTR258NzQ3ODA0OTg2MQ== (private=MTY4OTQwODc0N3x4eVlRWlhyX2hBSkdZc3o3STh5MWdkLWljd2NZd3VBOEtRd3JqNklrVXVUSl9SY2tJVWI4VTBPMWVRd1p6SFRaNUdWei05ckhiZTRmcjYyWHIyd2RwSG5VSEhXUHh6Rkh1cXFUanZJRnR6blpaQlRHVE9TNWxFUmVLclpKMl9PRjJwMFd0Zi1kLVpOQjIyUGNVY0pidXpJTG1BS0FqNUl2cFo2UlI2UFdfM2E5R2c9PXxzJ2xtd5GBvg2coYvE3zo8T-oZmto244z3vybk__WV1Q==)
Jul 15 09:14:40 turn signaling[2430]: client.go:301: Error reading from client aTX587IW_citz2tS2eNCYnAfwaQOp78nACqWAOJFPud8PT13TWQ5ZWpiRnZkRUthd3p4RUZmaDJkU08wNXhYOG9oeXJDekdpVTRvTDFsb0xEajZ3LXZHeGRxRXlFa0Rtc1NCdl9GV1FsZXVxQWRwME5ILTZrOV94b1FLdHBzUXpSYjZxSDd5bGpVd2p1S01oYWtPbXJnYlIzSlJCU1BWSWdBODFuU0JtYTYtUUxXaE53bFBVeUp0bzdmWGxwd015NElnYlNyZXVyYUFCU3JlSUd8NzQ3ODA0OTg2MQ==: websocket: close 1006 (abnormal closure): unexpected EOF
Jul 15 09:14:40 turn signaling[2430]: hub.go:832: Unregister aTX587IW_citz2tS2eNCYnAfwaQOp78nACqWAOJFPud8PT13TWQ5ZWpiRnZkRUthd3p4RUZmaDJkU08wNXhYOG9oeXJDekdpVTRvTDFsb0xEajZ3LXZHeGRxRXlFa0Rtc1NCdl9GV1FsZXVxQWRwME5ILTZrOV94b1FLdHBzUXpSYjZxSDd5bGpVd2p1S01oYWtPbXJnYlIzSlJCU1BWSWdBODFuU0JtYTYtUUxXaE53bFBVeUp0bzdmWGxwd015NElnYlNyZXVyYUFCU3JlSUd8NzQ3ODA0OTg2MQ== (private=MTY4OTQwODc0N3w1WDdUaHE0NGlwMlByQkZacldNRVZUeVhWMUpYdjZuWnZFeUpBbEppaS1fRFlJYUdNdW1odnJ6ME9XYXdpa2RKOG5Rajdpai1VcW03M1pyV1lvSkY0MXBDZW8zUjJCX1VaZ1J2ZzZwclI4RTA1WWJuY2FXaFVMUnQ4ZlFGd1QzR0ZiMmEzcmxUR1p1Ulp1N3BHcFFodGt2UDd0S1lFZ1B4TGRNUk5RbmJMbzN1SkE9PXzb5_GxRtxNXZs9I3gWy-V_6VZ7ROXPwo82Fhwonv8x_Q==)
Jul 15 09:15:10 turn signaling[2430]: hub.go:585: Closing expired session IF_gNVm-TZW9RaZ2-nH8I4vaI9mUtFCJZ-IYMUGuQSZ8PT1RZnBrZjVOVWVWMUs4YkRnV09WRFIxM1BBZ245anBHY3ZaV3QwTWdVbTE0ZXo3ak1hZGE3dHZtSVMtcDl6bkZlTTdVWTV5dnpmV0NLbXBlLVFXNGtZVmtNbTZQWF9fOGppUXAzUTctOXdTQzNubHIxU0ZyWWdnaW1SYUZHM3NLaVRfOWJKODZEdzR0d1FSQ2RuaXZEanJsLU1iTHN0dUhEM0ZNN2o0eDNEejFTR258NzQ3ODA0OTg2MQ== (private=MTY4OTQwODc0N3x4eVlRWlhyX2hBSkdZc3o3STh5MWdkLWljd2NZd3VBOEtRd3JqNklrVXVUSl9SY2tJVWI4VTBPMWVRd1p6SFRaNUdWei05ckhiZTRmcjYyWHIyd2RwSG5VSEhXUHh6Rkh1cXFUanZJRnR6blpaQlRHVE9TNWxFUmVLclpKMl9PRjJwMFd0Zi1kLVpOQjIyUGNVY0pidXpJTG1BS0FqNUl2cFo2UlI2UFdfM2E5R2c9PXxzJ2xtd5GBvg2coYvE3zo8T-oZmto244z3vybk__WV1Q==)
Jul 15 09:15:10 turn signaling[2430]: hub.go:585: Closing expired session aTX587IW_citz2tS2eNCYnAfwaQOp78nACqWAOJFPud8PT13TWQ5ZWpiRnZkRUthd3p4RUZmaDJkU08wNXhYOG9oeXJDekdpVTRvTDFsb0xEajZ3LXZHeGRxRXlFa0Rtc1NCdl9GV1FsZXVxQWRwME5ILTZrOV94b1FLdHBzUXpSYjZxSDd5bGpVd2p1S01oYWtPbXJnYlIzSlJCU1BWSWdBODFuU0JtYTYtUUxXaE53bFBVeUp0bzdmWGxwd015NElnYlNyZXVyYUFCU3JlSUd8NzQ3ODA0OTg2MQ== (private=MTY4OTQwODc0N3w1WDdUaHE0NGlwMlByQkZacldNRVZUeVhWMUpYdjZuWnZFeUpBbEppaS1fRFlJYUdNdW1odnJ6ME9XYXdpa2RKOG5Rajdpai1VcW03M1pyV1lvSkY0MXBDZW8zUjJCX1VaZ1J2ZzZwclI4RTA1WWJuY2FXaFVMUnQ4ZlFGd1QzR0ZiMmEzcmxUR1p1Ulp1N3BHcFFodGt2UDd0S1lFZ1B4TGRNUk5RbmJMbzN1SkE9PXzb5_GxRtxNXZs9I3gWy-V_6VZ7ROXPwo82Fhwonv8x_Q==)

</details>
it seems to double register the 17.0.1 and the v3177 on 203 when the 17.0.1 is def closed - ill go back to Andriod thanks once again in pointing me in the right direction

--- a/app/src/main/java/com/nextcloud/talk/webrtc/PeerConnectionWrapper.java
+++ b/app/src/main/java/com/nextcloud/talk/webrtc/PeerConnectionWrapper.java
@@ -398,8 +398,7 @@ public class PeerConnectionWrapper {
         @Override
         public void onStateChange() {
             if (dataChannel != null &&
-                dataChannel.state() == DataChannel.State.OPEN &&
-                "status".equals(dataChannel.label())) {
+                dataChannel.state() == DataChannel.State.OPEN) {
                 sendInitialMediaStatus();
             }
         }
@@ -535,10 +534,12 @@ public class PeerConnectionWrapper {
 
         @Override
         public void onDataChannel(DataChannel dataChannel) {
-            if ("status".equals(dataChannel.label()) || "JanusDataChannel".equals(dataChannel.label())) {
-                PeerConnectionWrapper.this.dataChannel = dataChannel;
-                PeerConnectionWrapper.this.dataChannel.registerObserver(new MagicDataChannelObserver());
+            if (PeerConnectionWrapper.this.dataChannel != null) {
+                Log.w(TAG, "Data channel with label " + PeerConnectionWrapper.this.dataChannel.label()
+                    + " exists, but received onDataChannel event for DataChannel with label " + dataChannel.label());
             }
+            PeerConnectionWrapper.this.dataChannel = dataChannel;
+            PeerConnectionWrapper.this.dataChannel.registerObserver(new MagicDataChannelObserver());
         }
 
         @Override

